python/python-core/control-flow-i/if-statements.md

```diff
@@ -43,27 +42,27 @@ The code above will print `'3 is a positive number'`.
 The `if` statement can be extended to include a *catch-all*, `else`, that will be executed if the condition is not met.
 
 ```python
-num = -1
-if num >= 0:
-    print("Positive or Zero")
+num = 1
+if num == 0:
+    print("Zero")
 else:
-    print("Negative number")
+    print("Positive number")
 ```
 
-The code above will print `'Negative number'`.
+The code above will print `'Positive number'`.
 
-> As you may have spotted, we can use *comparison operators* to create conditions in Python.
+> Note that the single `=` is used for assigning values to variables, and the double `==` for comparison. There are several different *comparison operators*.
```

Comment:
```suggestion
> 💡 A single `=` is used for assigning values to variables, and the double `==` for comparison. There are several different *comparison operators*.
```

Instead of saying the word "Note", we can use the 💡 emoji. Most of our notes have been converted to have an emoji on start. 💡, ❗, and so on, depending on the context of the note